VestingInstance.sol

```diff
@@ -0,0 +1,273 @@
+    event VestingAccountVoterAuthorized(address authorizer, address voter, uint256 timestamp);
+    event VestingAccountValidatorAuthorized(address authorizer, address validator, uint256 timestamp);
+    event VestingWithdrawn(address beneficiary, uint256 amount, uint256 timestamp);
+    event VestingRevoked(address revoker, address refundDestination, uint256 refundDestinationAmount, uint256 timestamp);
+
+    // total that is to be vested
+    uint256 public _vestingAmount;
+
+    // amount that is to be vested per period
+    uint256 public __vestAmountPerPeriod;
+
+    // number of vesting periods
+    uint256 public __vestingPeriods;
+
+    // beneficiary of the amount
+    address public _beneficiary;
+
+    // durations in secs. of one period
+    uint256 public _vestingPeriodSec;
+
+    // timestamps for start and cliff starting points. Timestamps are expressed in UNIX time, the same units as block.timestamp.
+    uint256 public _cliffStartTime;
+    uint256 public _vestingStartTime;
+
+    // indicates if the contract is revokable
+    bool public _revocable;
+
+    // revoking address and refund destination
+    address public _refundDestination;
+    address public _revoker;
+
+    // indicates how much of the vested amount has been released for withdrawal (i.e. withdrawn)
+    uint256 public _currentlyReleased;
+
+    // indicates if the vesting has been revoked. false by default
+    bool public _revoked;
+
+    // the time at which the revocation has taken place
+    uint256 public _revokeTime;
+
+    /**
+     * @notice A constructor for initialising a new instance of a Vesting Schedule contract
+     * @param beneficiary address of the beneficiary to whom vested tokens are transferred
+     * @param vestingAmount the amount that is to be vested by the contract
+     * @param vestingCliff duration in seconds of the cliff in which tokens will begin to vest
+     * @param vestingStartTime the time (as Unix time) at which point vesting starts
+     * @param vestingPeriodSec duration in seconds of the period in which the tokens will vest
+     * @param vestAmountPerPeriod the vesting amound per period where period is the vestingAmount distributed over the vestingPeriodSec
+     * @param revocable whether the vesting is revocable or not
+     * @param revoker address of the person revoking the vesting
+     * @param refundDestination address of the refund receiver after the vesting is deemed revoked
+     * @param vestingContractOwner the owner that of the vesting contract
+     */
+    constructor (address beneficiary,
+                uint256 vestingAmount,
+                uint256 vestingCliff,
+                uint256 vestingStartTime,
+                uint256 vestingPeriodSec,
+                uint256 vestAmountPerPeriod,
+                bool    revocable,
+                address revoker,
+                address refundDestination,               
+                address vestingContractOwner) public {
```

Comment:
Do we need this variable. Can you think of any special privileges for this address? I could not think of any

Reply:
not really, i have removed it